### DISCUSSION TITLE
Installation?

### DISCUSSION THREAD

**Discussion by Raeemuth** [sukalaper/dotfiles#9] [Q&A] [answered]
I'm new here, maybe you can help with the installation steps? 

Or is there a reference I can read?😁

**Comment by sukalaper** [comment, marked as answer]
> I'm new here, maybe you can help with the installation steps?
> 
> Or is there a reference I can read?😁

Ah, certainly. Apologies for the oversight in documentation. Firstly, ensure that you have `git` installed. 

Then, clone this repository. 
```
$ git clone https://github.com/sukalaper/dotfiles.git
```

Copy all or some of the configuration into your `~/.config` directory. Make edits according to your preferences. Restart your machine (optional). With this, you should be ready.
